release-please-config.json

```diff
@@ -0,0 +1,16 @@
+{
+    "packages": {
+      "gitops": {
```

Comment:
This one will actually need to be "." as pyproject.toml is located at the top level.

We'll need some excludes to gitops server now that I think about it.

I also think we might be able to skip `gitops_server`. We can have `gitops_server` building on all pushes to main.

Reply:
Sorry for the round trip; just given it more thought :(